library/src/components/connectors/destination/select-destination/SelectDestination.vue

```diff
@@ -218,4 +286,10 @@ export default {
 
 <style lang="sass" scoped>
 @import '../../../../styles/mixins/connectors/destination/select-destination/_select-destination'
+
+.control
+  .filter-value
+    .nio-text-field.small-input
+      min-width: 140px
+      width: 25% !important
```

Comment:
@woodypowr5 Forgive my sin here, was having trouble getting a high enough level of specificity to override the `width: 100%` in `_select-destination.sass`. LMK if you know of a better way off the top of your head.